datalad/resources/website/assets/js/main.js

```diff
@@ -78,6 +78,22 @@ function getParameterByName(name, url) {
 }
 
 /**
+ * Create Breadcrumbs to current location in dataset
+ * @return {array} html linkified breadcrumbs array
+ */
+function bread2crumbs() {
+  var raw_crumbs = loc().href.split('/');
+  var crumbs = [];
+  for (var index = 2; index < raw_crumbs.length() - 1; index++) {
+    if (raw_crumbs[index] !== '?dir=')
+      continue;
```

Comment:
So I guess now we need to wait for you to finish it up?
As for annex vs git vs directory - just use colors as within the table. I did find a few new issues discovered when I finally deployed the beast! Looks neat ;-)